src/tests/ftest/util/dmg_utils.py

```diff
@@ -721,20 +748,39 @@ def storage_query_blobstore(self, devuuid, tgtid=None):
         """
         self.set_sub_command("storage")
         self.sub_command_class.set_sub_command("query")
+        self.sub_command_class.sub_command_class.set_sub_command("list-pools")
         self.sub_command_class. \
-            sub_command_class.set_sub_command("blobstore-health")
+            sub_command_class.sub_command_class.rank.value = rank
+        return self._get_result()
+
+    def storage_query_device_health(self, uuid):
+        """Get the result of the 'dmg storage query device-health' command.
+
+        Args:
+            uuid (str): Device UUID to query.
+
+        Returns:
+            CmdResult: an avocado CmdResult object containing the dmg command
+                information, e.g. exit status, stdout, stderr, etc.
+
+        Raises:
+            CommandFailure: if the dmg storage prepare command fails.
+
+        """
+        self.set_sub_command("storage")
+        self.sub_command_class.set_sub_command("query")
         self.sub_command_class. \
-            sub_command_class.sub_command_class.devuuid.value = devuuid
+            sub_command_class.set_sub_command("device-health")
         self.sub_command_class. \
-            sub_command_class.sub_command_class.tgtid.value = tgtid
+            sub_command_class.sub_command_class.uuid.value = uuid
         return self._get_result()
 
-    def storage_query_device_state(self, devuuid):
-        """Get the result of the 'dmg storage query device-state' command.
+    def storage_query_target_health(self, rank, tgtid):
```

Comment:
Are both of the args required for the command to execute?

Reply:
Yes, because target without rank is ambiguous (the same target id can be valid on more than one rank per server).